src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -45,24 +46,36 @@ const NewProjectView = ({
   setProjectName,
 }: Props) => {
   const dispatch = useDispatch();
+  const [isAwaitingProjectName, setIsAwaitingProjectName] =
+    useState<boolean>(true);
```

Comment:
Do we need an explicit type here?

Reply:
As in using `useState(true)` instead of `useState<boolean>(true)`? If both work then it we don't need it. I thought you always had to have the type.

I'll change it soon

Reply:
Here it is: [8ba7171](https://github.com/chloebrett/mlvet/pull/175/commits/8ba7171f6275fac423d921762e10a711e54fb490)